pypeit/core/skysub.py

```diff
@@ -113,9 +115,15 @@ def global_skysub(image, ivar, tilts, thismask, slit_left, slit_righ, inmask=Non
 
     # Init
     (nspec, nspat) = image.shape
-    piximg = tilts * (nspec-1)
+    if use_wave:
```

Comment:
Please remove this argument and this code. You can simply deal with this in the calling code by creating a psuedo tilt image, i.e. 

tilt_psuedo = (waveimg - wavimg.min())/(waveimg.max()-waveimg.min()) 

There is a good reason why we use an image in pixel coordinates as the x-coordinate for sky-subtraction, because then the break-point spacing bsp is in pixels.  I'm not sure if you dealt with this in your code or just got lucky somehow. But I would avoid doing this fit in any other coordinates besides tilts*(nspec-1).

Reply:
Great suggestion - I've implemented this!